src/main/java/frc/robot/util/CompoundCommands.java

```diff
@@ -277,15 +299,15 @@ public static DeferredCommand deferDrive(Supplier<Command> command) {
   public static Command armToGoal(double wristSetPos, double pivotSetPos, double elevatorSetPos) {
 
     if (elevatorSetPos > elevator.getPosition().getAsDouble()) {
-      return pivot
-          .setPosition(() -> pivotSetPos)
-          .andThen(elevator.setPosition(() -> elevatorSetPos))
-          .alongWith(wrist.setGoalPosition(() -> wristSetPos));
-
-    } else {
       return elevator
           .setPosition(() -> elevatorSetPos)
-          .alongWith(wrist.setGoalPosition(() -> wristSetPos))
+          .andThen(wrist.setGoalPosition(() -> wristSetPos))
+          .andThen(pivot.setPosition(() -> pivotSetPos));
+
+    } else {
+      return wrist
```

Comment:
Assuming pivot is already in a safe position, the same optional potential improvement from above, moving elevator and wrist at same time.